app/graphql/mutations/pairings/create_pairings.rb

```diff
@@ -0,0 +1,17 @@
+module Mutations
+  module Pairings
+    class CreatePairings < ::Mutations::BaseMutation
+      argument :pairer_id, Integer, required: true
+      argument :pairee_id, Integer, required: true
+      argument :date, String, required: true
+      argument :time, String, required: true
+      argument :notes, String, required: false
```

Comment:
Matt, nice work putting this together. I noticed that there's a difference between how you wrote the mutation's arguments and what the FE has in its query, which might be related to something we figured out yesterday. Our initial assumption was that pairings were created and saved to the database when someone clicks "Book Session," but we discovered yesterday that the actual case is that the pairings are created with `pairee` as `null` when a user defines their availability, and then they are _updated_ when another user books a session with the pairer. So the mutation in `src/queries/index.js` is like this:
```ruby
mutation {
    createPairings(
      pairings: ${pairingsString}
    ) {
      pairer
      date
      time
    }
  }
```
Which doesn't have a `pairee_id` or `notes` as arguments. Can you revise it accordingly?